andela/travel_tool_back
# #160597069 Check available room and bed space during new travel request

#### What does this PR do?

This PR ensure that user enters available room and bed space during new travel request creation.

#### Description of Task to be completed?

As a user, I should be able to add the room and bed space during my new travel request. This will ensure that if I will not get any room available, then will look for alternatives or postpone trip duration. 

#### How should this be manually tested?
- git clone https://github.com/andela/travel_tool_back.git
- git checkout ft-check-available-room-bed-160597069
- yarn install && yarn run db:rollmigrate
- In the user table, assign yourself travel admin role by changing id to 29187
- start the app
- Open postman and provide your token in the authorization header. Enter url POST http://localhost:5000/api/v1/guesthouses
- add [this](https://gist.github.com/ezrqnkemboi/cb995fdec424882b155f4d00c24d1a54) data as json for post request

#### Any background context you want to provide?
Ensure that before creating request, you should create a guest house using [this PR](https://github.com/andela/travel_tool_back/pull/65)

#### What are the relevant pivotal tracker stories?

[#160597069](https://www.pivotaltracker.com/story/show/160597069)

#### Screenshots (if appropriate)


#### Questions:
